code/__HELPERS/lordcolor.dm

```diff
@@ -51,4 +51,13 @@ GLOBAL_VAR(lordsecondary)
 		GLOB.lordcolor -= O
 	for(var/turf/T in GLOB.lordcolor)
 		T.lordcolor(prim,sec)
-		GLOB.lordcolor -= T
\ No newline at end of file
+		GLOB.lordcolor -= T
+
+//	Default Kingdom colors, if none are picked.
+proc/lord_color_default()
+	GLOB.lordprimary = "#933030" //RED
+	GLOB.lordsecondary = "#2f352f" //BLACK
+	for(var/obj/O in GLOB.lordcolor)
+		O.lordcolor(GLOB.lordprimary,GLOB.lordsecondary)
+	for(var/turf/T in GLOB.lordcolor)
+		T.lordcolor(GLOB.lordprimary,GLOB.lordsecondary)
```

Comment:
Is this a lot of turfs? If so this could be expensive(and produce a short lagspike)

Reply:
I am pretty sure this is just the throneroom carpet, which isn't a lot.